src/main/groovy/cz/alenkacz/gradle/scalafmt/ScalafmtFormatBase.groovy

```diff
@@ -3,30 +3,37 @@ package cz.alenkacz.gradle.scalafmt
 import org.gradle.api.DefaultTask
 import org.gradle.api.plugins.JavaBasePlugin
 import org.gradle.api.tasks.SourceSet
-import org.scalafmt.Scalafmt
+import org.scalafmt.interfaces.Scalafmt
+import org.scalafmt.interfaces.ScalafmtClassLoader
+import java.nio.file.Paths
 
 class ScalafmtFormatBase extends DefaultTask {
     SourceSet sourceSet
-
+    ClassLoader cl = this.class.getClassLoader()
     PluginExtension pluginExtension
+    def formatter = Scalafmt.create(cl)
+            .withRespectVersion(false)
+            .withDefaultVersion("1.5.1")
```

Comment:
I don't like this setting being so deep in the code. I am used to looking at dependencies - I think it can be puzzling if you see 2.0 but you're somehow not getting 2.0 features.

Thinking what would be a good solution here.
- not having this default here at all
- exposing it somewhere in some config or settings
- ?

I am kind of inclined to go with first option and not override this version here and let it up to users to override if they need. What do you think?

Reply:
I have added this because i didn't want to break compatibility with the previous version. I.e. the current version defaults to 1.5.1.

Otherwise you have to always set the version explicitly, so now if you set the version it overrides the fmt version used.
https://scalameta.org/scalafmt/docs/configuration.html#other

so it is already as you asked. but you have to set it inside the config.

Reply:
I thought about it while walking from gym :) let's go with what you proposed. When 2.0 is stable, I'll release new 2.0 version of the plugin and will drop this `withDefaultVersion` and then it will be again in sync (the version in build.gradle and the default version)